examples/ecs/command_error_handling.rs

```diff
@@ -0,0 +1,74 @@
+use bevy::{core::FixedTimestep, prelude::*};
+
+fn main() {
+    App::build()
+        .add_startup_system(setup.system())
+        .add_system_set(
```

Comment:
You should just be able to add the run criteria directly to your system these days :)